src/js/models/markup.js

```diff
@@ -10,27 +11,36 @@ export const VALID_MARKUP_TAGNAMES = [
   'li'
 ].map(normalizeTagName);
 
+export const VALID_ATTRIBUTES = [
```

Comment:
I see this is used as a filter during parsing, but should it also be used for an assertion here as `VALID_MARKUP_TAGNAMES` etc is? IMO yes.